Phoenix/hello/lib/hello_web/templates/hello/show.html.heex

```diff
@@ -0,0 +1,3 @@
+<section class="phx-hero">
+    <h2>Hello World, from <%= @messenger %>!</h2>
+</section>
```

Comment:
@samu975 Again, please configure your text editor to automatically add a blank line at the end of the files

Reply:
I already did, maybe these files was created before of config the vs code. Sorry about that